### ISSUE TITLE
Map has leaked in the void

### ISSUE THREAD

**Issue by User55555555555** [BEEmod/BEE2.4#1956]
### BEE2 Application Version

2.4.43.0 64bit

### BEE2 Package Version

v4.42.0

### What operating system are you running on?

Windows 11

### Description of the bug

When I compile my game it gives an error and says my map has a leak, the leak appears outside of the walls of the test chamber.

### Expected behavior

The test chamber should compile and let me play it.

### Steps to reproduce

I tried but couldn't reproduce the bug.

### Logs and Reports

```shell
This map has "leaked". This is a bug in an item or style, which should be fixed. The displayed line indicates the location of the leak, you may be able to resolve it by removing/modifying items in that area. Please submit a bug report to the author of the item so this can be resolved. Leak coordinates:
(512 -512 -1328)
(3e-06 -16 -16)
```

### Additional information

 I did not use any BEEMOD items in this test chamber at all, Clean style, GLaDOS Voice, Skybox Clean.
![Screenshot_72](https://github.com/BEEmod/BEE2.4/assets/135179815/b2153015-d755-4740-a28c-5a3c95bd5491)

**Comment by Luke18033** [comment]
If you're using any items from user created packages, try removing them from the map to see if they're the cause of the leak. Otherwise, if the issue persists, go to `Portal 2/sdk_content/maps/styled`, put `preview.vmf` and `preview.lin` in a ZIP, then upload that here.

**Comment by User55555555555** [comment]
I used the default items and no user created packages.

**Comment by Luke18033** [comment]
Can you try and rebuild the map again, then send those files I mentioned?

**Comment by User55555555555** [comment]
[preview.error.zip](https://github.com/BEEmod/BEE2.4/files/11648221/preview.error.zip)

Is the solution just to open the map in hammer and drawing a block on top?

**Comment by Luke18033** [comment]
No, that's not how you should fix leaks. If you do "load pointfile" it will show a line going through where the leak is so you can fix it. But BEE maps shouldn't be able to leak in the first place, so if that is possible it's considered a bug.

**Comment by Luke18033** [comment]
Can you upload `preview.log` too?

**Comment by User55555555555** [comment]
[preview.log](https://github.com/BEEmod/BEE2.4/files/11711491/preview.log)

**Comment by Luke18033** [comment]
Try going in the BEE2 app settings, clicking "reset package cache", then re-exporting, and see if it's still broken.

**Comment by mrkittenisdabest** [comment]
i had the same problem. and the reset package cache and re-exporting did not work, is there any other way to fix it?
